android/src/toga_android/widgets/optioncontainer.py

```diff
@@ -129,14 +129,16 @@ def add_option(self, index, text, widget, icon=None):
         if index >= self.max_items:
             warnings.warn(
                 f"OptionContainer is limited to {self.max_items} items on "
-                "Android. Additional item will be ignored."
+                "Android. Additional item will be ignored.",
+                stacklevel=2,
```

Comment:
Preference here would be to put brackets around the string, so this doesn't visually present as 3 arguments.